templates/emails/group.expense.paid.hbs

```diff
@@ -0,0 +1,37 @@
+Subject: Expense paid on {{group.name}}: {{{currency expense.amount currency=expense.currency}}} for {{expense.title}} by {{user.username}}
+
+{{> header}}
+
+<style>
+.btn {
+  color: #fff;
+  background: #80d823;
+  border: 1px solid transparent;
+  border-radius: 28px;
+  display: inline-block;
+  padding: .75rem 1rem;
+  text-transform: uppercase;
+  text-align: center;
+  font-weight: 700;
+  min-width: 150px;
+  box-sizing: border-box;
+  vertical-align: middle;
+}
+
+.btn.red {
+  background: #dd2663;
+}
+</style>
+
+<center>
+  <h3> Hi there! Just a note to let you know that an expense was paid in {{group.name}}. </h3>
```

Comment:
That's the eventual goal. That requires different templates - we'll do it in another PR. Let's get this merged to fix current emails with broken images and missing subject.